front/app/components/admin/ActionForm/index.tsx

```diff
@@ -22,26 +22,33 @@ import GroupSelect from './GroupSelect';
 import messages from './messages';
 import { showResetButton } from './utils';
 
+type Changes = {
+  permittedBy?: PermittedBy;
+  groupIds?: string[];
+  verificationExpiry?: number | null;
+};
+
 interface Props {
   phaseId?: string;
-  groupIds?: string[];
   permissionData: IPermissionData;
-  onChange: (permittedBy: PermittedBy, groupIds: Props['groupIds']) => void;
+  onChange: (changes: Changes) => void;
   onReset: () => void;
 }
 
-const ActionForm = ({
-  phaseId,
-  permissionData,
-  groupIds,
-  onChange,
-  onReset,
-}: Props) => {
+const ActionForm = ({ phaseId, permissionData, onChange, onReset }: Props) => {
   const {
```

Comment:
We've had issues with this type of restructuring in the past where our types say these attributes are always there but it's not the case. I know it was not introduced in this PR but just pointing it out to make extra sure nothing crashes because of this.

Reply:
Seems like a type issue, not a destructuring issue 😉 I'm pretty confident about these types so I will leave it